experimenter/experimenter/experiments/migrations/0249_alter_nimbusexperiment_application.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.2.20 on 2023-08-22 22:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
All migrations should be squashed together. What changes caused this migration to be generated?

Reply:
I updated merged the main branch into this one and it suggested to run `make migrations` when I ran `make check`